pkg/shell/shell-modals.jsx

```diff
@@ -46,14 +46,16 @@ export class AboutCockpitModal extends React.Component {
     componentDidMount() {
         const packages = [];
 
-        cockpit.spawn(["bash", "-ec", "command -v zypper dnf apt | head -n1 | xargs --no-run-if-empty basename"], [], { err: "message" })
+        cockpit.spawn(["bash", "-ec", "command -v zypper dnf apt pacman | head -n1 | xargs --no-run-if-empty basename"], [], { err: "message" })
```

Comment:
How about we completely ditch this `command` thing, and just run `(set +e; rpm -qa ... ; dpkg-query ..; pacman -Q ...) | grep cockpit | sort` ? This avoids the "zypper" vs. "yum" vs. "dnf" knowledge and has fewer moving parts.

With a lot of effort you can probably construct a system where you install cockpit-system.deb and cockpit-podman.rpm, but (1) it's actually correct to show both packages in the list, and (2) who actually does that :grin:

Reply:
Or rather, drop the `-e` shell option and skip the `set +e` of course.

Reply:
Makes sense, applied!